runtime/altair/src/lib.rs

```diff
@@ -1666,13 +1741,16 @@ construct_runtime!(
 		BlockRewards: pallet_block_rewards::{Pallet, Call, Storage, Event<T>, Config<T>} = 105,
 		Keystore: pallet_keystore::{Pallet, Call, Storage, Event<T>} = 106,
 		PriceCollector: pallet_data_collector::{Pallet, Storage} = 107,
+		LiquidityPools: pallet_liquidity_pools::{Pallet, Call, Storage, Event<T>} = 108,
+		LiquidityPoolsGateway: pallet_liquidity_pools_gateway::{Pallet, Call, Storage, Event<T>, Origin } = 115,
```

Comment:
Note that I kept it `115` like on development since we had spoken, in the last few weeks, about trying to keep pallet indexes the same across the three runtimes. I am fine making it `109` so that's sorted within the `altair` runtime but then it won't be consistent cross runtimes. wdyt? cc @mustermeiszer

Reply:
Can we even make the ordering the same on all three? If so, I would like to keep that, if not, lets have sequential.

Reply:
We sure could try but let's not spend time on this now. Made it `109` to be sequential 👍